tools/local-env/scripts/docker.js

```diff
@@ -7,5 +7,10 @@ dotenvExpand.expand( dotenv.config() );
 
 const composeFiles = local_env_utils.get_compose_files();
 
+if (process.argv.includes('--coverage-html')) {
+	process.env.LOCAL_PHP_XDEBUG = 'true';
```

Comment:
I haven't tested to confirm, but I don't think this will work as is. Just changing the environment variables has no effect unless the container is stopped and restarted because these are read on environment start up.

Reply:
I wasn't sure, but it seems to work

Reply:
Ah, yes. Confirming that this does work. The reason is that `test:php` technically starts a separate PHP container than any that is already started. So whether the original PHP container has xDebug enabled does not matter.